src/impl_/pymodule.rs

```diff
@@ -217,6 +218,17 @@ impl PyAddToModule for ModuleDef {
     }
 }
 
+pub fn add_const_to_module<'py, T: IntoPyObject<'py>>(
+    m: &Bound<'py, PyModule>,
+    name: &CStr,
+    value: T,
+) -> PyResult<()> {
+    m.index()?.append(name.to_str()?)?;
+    let value = value.into_pyobject(m.py()).map_err(Into::into)?.into_ptr();
+    let res = unsafe { ffi::compat::PyModule_Add(m.as_ptr(), name.as_ptr(), value) };
+    error_on_minusone(m.py(), res)
```

Comment:
Can we just `m.add()` here? It allocates a `PyString` for `name`, so there's no downsid?

Reply:
True, initially i didn't have the `index` bit, but now there is no advantage of `PyModule_Add` anymore. I guess the whole helper is not needed anymore then, we can simply emit the `m.add` call directly. I will simplify it.